src/lib/transaction/operations/DynamicOperationComponentFactory.ts

```diff
@@ -102,6 +103,9 @@ export default class DynamicOperationComponentFactory {
                     operation as Operation.RevokeOfferSponsorship,
                 );
                 break;
+            case 'clawbackClaimableBalance':
+                operationComponent = new ClawbackClaimableBalanceComponent(tx, operation);
```

Comment:
Why do some operations get casted and others don't?

Reply:
revokeSponsorship is kind of the default revoke operation, but there are many derived revoke operations from it. I need to do this to solve all problems with TS.